specs/protocol/configurability.md

```diff
@@ -0,0 +1,79 @@
+**Table of Contents**
+
+- [Consensus Parameters](#consensus-parameters)
+- [Policy Parameters](#policy-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are four categories of OP Stack configuration options:
+
+- **Consensus Parameters**: Parameters and properties of the chain that may
+  be set at genesis and fixed for the lifetime of the chain, or may be
+  changeable through a privileged account.
+- **Policy Parameters**: Parameters of the chain that might be changed without
+  breaking consensus. Consensus is enforced by the protocol, while policy parameters
+  may be changeable within constraints imposed by consensus.
+- **Admin Roles**: These roles can upgrade contracts, change role owners,
+  or update protocol parameters. These are typically cold/multisig wallets not
+  used directly in day to day operations.
+- **Service Roles**: These roles are used to manage the day-to-day
+  operations of the chain, and therefore are often hot wallets.
+
+## Consensus Parameters
+
+| Config Property                       | Description                                                                                                                  | Administrator                       |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| [Batch Inbox address](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L176)                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                  | [L1 Proxy Admin](#admin-roles)                      |
+| [Chain ID](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/chainids.json)                              | Unique ID of Chain used for TX signature validation.                                                                         |                                     |
+| [Challenge Period](https://github.com/ethereum-optimism/superchain-registry/pull/44)                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                        | [L1 Proxy Admin](#admin-roles)                      |
+| Data Availability                     | Where L2 transaction calldata is posted.                                                                                     |                                     |
```

Comment:
```suggestion
```
Can we delete this entry? The DA layer is not configurable currently other than call vs blobdata, which I'll leave another comment on